### PR TITLE
Add docker-compose file to setup testing environment

### PR DESCRIPTION
**Pull request by jonathangreen**
## Description

Add a small docker-compose file that builds and runs containers to quickly get setup with a working circulation manager from the code in the local repository.

## Motivation and Context

Similar to this PR:
https://github.com/ThePalaceProject/virtual-library-card/pull/130

And the `docker-compose` file included with the library registry code.

Since we discussed in the sprint retrospective, the need to more easily setup an environment for things like code review, this PR attempts to make that process a little easier. I needed to setup a dockerized environment for testing https://github.com/ThePalaceProject/circulation/pull/576. So I took the opportunity to write a basic docker-compose file.

## How Has This Been Tested?

I've been using it to test https://github.com/ThePalaceProject/circulation/pull/576

## Checklist

- [X] I have updated the documentation accordingly.
- [X] All new and existing tests passed.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      POSTGRES_DB: circ
+
+  minio:
+    image: "bitnami/minio:2022.3.3"
+    ports:
+      - "9000:9000"
+      - "9001:9001"
+    environment:
+      MINIO_ACCESS_KEY: "palace"
+      MINIO_SECRET_KEY: "test123456789"
```
Comment (io7m) [inline docker-compose.yml:37]: I get a big warning here:

```
circulation-minio-1    | WARNING: MINIO_ACCESS_KEY and MINIO_SECRET_KEY are deprecated.
circulation-minio-1    |          Please use MINIO_ROOT_USER and MINIO_ROOT_PASSWORD
```

**Review by io7m — CHANGES_REQUESTED** [review]
I'm able to run this on Linux, which I didn't expect (given what happened the last time I ran `docker-compose` here!). It appears to work correctly, but there's a fairly loud deprecation warning.

File: docker-compose.yml (hunk 1 above)
Comment (jonathangreen) [reply docker-compose.yml]: 👍🏻 Didn't even notice those warnings 😅

**Review by io7m — APPROVED** [review]
Looks good! :rocket:

**Pull request merged**